design-reviews/recipes/feature-story-design-review-template.md

```diff
@@ -0,0 +1,60 @@
+# Your Feature or Story Design Title Here
+
+> Does the feature re-use or extend existing patterns / interfaces that have already been established for the project?
+> Does the feature expose new patterns or interfaces that will establish a new standard for new future development?
+
+* [Feature/Story Name](http://link-to-feature-or-story-work-item)
+* Engagement: [Engagement]
+* Customer: [Customer]
+* Authors: [Author1, Author2, etc.]
+
+## Overview/Problem Statement
```

Comment:
is this going to be a full-fledged design or is it an implementation level design for a given component?

Reply:
This is going to be more detailed to an implementation level since epic/milestone would be higher level than the story/feature level design.